.gitignore

```
@@ -66,10 +66,11 @@ node_modules/
 __pycache__/
 
 # VSCode
-.vscode/*
+.vscode/*/**
```

```suggestion
.vscode
```
Excluding a directory also excludes it's sub-directories.